src/pages/partners/payment-agent/_who-can-apply.js

```diff
@@ -23,7 +23,13 @@ import Reply from 'images/svg/pa-reply.svg'
 import Listed from 'images/svg/pa-listed.svg'
 
 const HowToApply = styled.div`
-    margin-top: 4 rem;
+    @media (max-width: 1024px) {
```

Comment:
We have the equivalent of `max-width: 1024px` which is `laptop`. You can take a look at `device.js` to see all available breakpoints.